neuralmonkey/config/configuration.py

```diff
@@ -67,11 +63,40 @@ def load_file(self, path):
             log("Failed to load INI file: {}".format(exc), color='red')
```

Comment:
.. a jestli to chceš mít tady, tak bys to měl dát s pozměněnou hláškou i do `build_model`, kde platí stejný argumenty jako tady.